cmd/brew-bootstrap-developer-system

```diff
@@ -2,35 +2,40 @@
 set -e
 
 # Setup shared dotfiles
-echo "Fetching cultureamp/web-team-dotfiles from GitHub…"
-if [ ! -d "$HOME/.dotfiles-web-team" ]; then
-  echo "Cloning to ~/.dotfiles-web-team:"
-  git clone $Q "https://github.com/cultureamp/web-team-dotfiles" ~/.dotfiles-web-team
+read -r -n 1 -p "Would you like install dotfiles (Y|n)?" answer
```

Comment:
```suggestion
read -r -n 1 -p "Would you like install dotfiles from https://github.com/cultureamp/web-team-dotfiles (Y/n)?" answer
```

Perhaps the URL here just to add a little more context for folk as to what they are electing to do?

I wonder also if we should make the variable `install_dotfiles` rather than `answer`, just in case more questions get added at some point?

Reply:
Actually that is a really good suggestion! 

Though should I add which dotfiles are important? The rationale behind this decision was because the dotfiles repo hasn't been updated in while, some dotfiles were more generic in nature, and some individuals have a specific needs when organising their dotfiles. 

Though talking about dotfiles could open a huge can of worms, so would it be best if we have a separate conversation about this? Perhaps do an audit? E.g should gitconfig be set at a project level or global level?